docs/BuildFromSource.md

```diff
@@ -225,6 +225,12 @@ TargetOsName             | The base runtime identifier to build for (win, linux,
 After building ASP.NET Core from source, you will need to install and use your local version of ASP.NET Core.
 See ["Artifacts"](./Artifacts.md) for more explanation of the different folders produced by a build.
 
+Building installers does not run as part of `build.cmd` run without parameters, so you should opt-in for building them: 
+
+```ps1
+build.cmd -BuildInstallers
```

Comment:
@jkotalik does `-buildInstallers` work w/o first building the x64 or x86 shared runtimes and the site extensions?

Reply:
Good point, some of the installers won't build without the x64 and x86 runtimes. I don't think we need site extensions at all, why do you suggest we need that?

Either way, you can add `build.cmd -all -pack -arch x64` and `build.cmd -all -pack -arch x86` before running it with -BuildInstallers

Reply:
I suggested the site extensions might be needed only because that's the order ci.yml builds things and I thought `-buildInstallers` included building an installer wrapping the site extensions.

In any case, @kant2002 could you add the following to these steps:
```ps1
.\build.cmd -all -pack -arch x64
.\build.cmd -all -pack -arch x86 -noBuildJava
```

A comment stating `-buildInstallers` and additional build steps aren't necessary on Linux or macOS may also help.